tests/spec/fs.copyFile.spec.js

```diff
@@ -0,0 +1,52 @@
+var util = require('../lib/test-utils.js');
+var expect = require('chai').expect;
+
+describe('fs.copyFile', function(){
+  beforeEach(function(done){
+    util.setup(function() {
+      var fs = util.fs();
+      fs.writeFile('/srcfile', 'This is a src file.', function(error){
+        if(error) throw error;
+        fs.writeFile('/destfile', 'This is a dest file.', function(error){
```

Comment:
I would recommend isolating the creation of the `/destfile` to the test case that requires it to limit global state in the tests, especially if it's only used in one test. If it becomes necessary for multiple test cases, we can always move it back into the `beforeEach()` step.

Reply:
Agree, please move all these `writeFile` calls to the individual tests that need them.  It's best if tests have as few pre conditions as possible, and everything gets done in the test itself.